code/scripting/lua/LuaValue.h

```diff
@@ -159,6 +159,21 @@ class LuaValue {
 		}
 	}
 
+	/**
+	 * @brief Gets the value of a userdata type using the passed getter function or throws an exception
```

Comment:
The comment is outdated now.
```suggestion
	 * @brief Same as above but allows passing a reference to a value where to store the lua value
```